space-plugins/nuxt-base-ui/components/ModalConfirm.vue

```diff
@@ -0,0 +1,42 @@
+<script setup lang="ts">
+defineProps<{
+	title?: string;
+	message?: string;
+}>();
+
+const emit = defineEmits(['confirmed']);
+
+const modalConfirmRef = ref<InstanceType<typeof BaseModal>>();
+
+const show = () => modalConfirmRef.value?.show();
+const close = () => modalConfirmRef.value?.close();
+
+defineExpose({ show, close });
+
+const onConfirmClicked = () => {
+	emit('confirmed');
+	close();
+};
+</script>
+
+<template>
+	<BaseModal ref="modalConfirmRef" :title="title" :message="message">
+		<template #actions>
+			<menu class="modal-action px-8 py-6 m-0 bg-gray-100">
+				<button
+					class="btn btn-tertiary"
+					title="Close the modal and cancel the action"
+				>
+					Cancel
```

Comment:
Maybe will be nice if you add a `cancel` emit, with that we can reuse this in some scenarios in future plugins.

Reply:
That's a good point, @ademarCardoso 💯 

By default a button within a `Dialog` automatically closes it, but having a proper event being emitted may be more clear and flexible.

Thanks for the hint 🙏 

Done here https://github.com/storyblok/space-tool-plugins/pull/79/commits/9039f0f66d10865a3ea1d117fc68ba79aece8d34